de_de updated for 3.2 and 3.2.1, waiting for 3.3

myron-obsolete-account opened the pull request:
Spent my time on a german translation for the Wither Storm mod
[de_de.txt](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/10096452/de_de.txt)
for german gamers

nonamecrackers2 committed:
2.0 changes

nonamecrackers2 committed:
2.0 additions

nonamecrackers2 committed:
2.0 additions

Loverxzc committed:
2.0 translate

Parallel011 committed:
Update zh_cn translate.

Fixed some translation, Also added new items translate to match en_us.

Parallel011 committed:
Add files via upload

nonamecrackers2 committed:
Merge pull request #166 from Parallel011/1.18.2

Update zh_cn translation

nonamecrackers2 committed:
Merge pull request #159 from Loverxzc/patch-1

2.0 ru_ru translation

nonamecrackers2 committed:
Update feature_request.md

Loverxzc committed:
2.0 translate

Some correction and adaptive

nonamecrackers2 committed:
Merge pull request #557 from Loverxzc/patch-2

2.0 ru_ru translation fixes

nonamecrackers2 committed:
2.1

nonamecrackers2 committed:
More translations

Spanish by pox1016 and Ukrainian by SaiDGaming)

nonamecrackers2 committed:
3.1

Parallel011 committed:
Update zh_cn translate.

Finished!
Added more translations to match newest en_us!

nonamecrackers2 committed:
Merge pull request #682 from Parallel011/1.19

Updated the zh_cn translation.

Parallel011 committed:
Update more Chinese translate.

I also made some translations for hk and tw.

Parallel011 committed:
a little fixed.

nonamecrackers2 committed:
Merge pull request #734 from Parallel011/1.19

Update more Chinese translate.

Luisinhi010 committed:
Create pt_br.json

Luisinhi010 committed:
Update pt_br.json

its pretty much finished the only thing i didn't do is the settings descriptions

Luisinhi010 committed:
I'm Blind

Luisinhi010 committed:
Now i finished it for real

Luisinhi010 committed:
making more consistent with original minecraft lang

nonamecrackers2 committed:
Merge pull request #889 from Luisinhi010/Pt-br

Brazilian Portuguese translation

myron-obsolete-account commented:
I will probably never understand pull requests.

nonamecrackers2 commented:
That's fine, this works

myron-obsolete-account commented:
Oh but I do need to add translations for 2.2/3.2, right? Right. Well crap. But I'd be glad to do it anyways.

myron-obsolete-account commented:
Okay, most likely the only place I can ask this, but is the new (OH MY GOD THE NEW 3.2 UPDATE RELEASED (duh)) 1.19.3 version compatible with 1.19.2? doesn't exactly look like it I guess, haven't tried it tho

nonamecrackers2 commented:
> Oh but I do need to add translations for 2.2/3.2, right? Right. Well crap. But I'd be glad to do it anyways.

You will, but there isn't a rush. Your files will still work just the new lang won't be translated. Closing also since this has been added

myron-obsolete-account commented:
:(

myron-obsolete-account commented:
Hey Crackalacka (sorry) or anyone else can you please reopen this because I can't create a new pull request for some reason and I am working on the updated translation, thanks.

Also one question, does the en_us file [in here](https://github.com/nonamecrackers2/crackers-wither-storm-mod/blob/1.19.4/lang/en_us.json) include the new translations aswell? or should I just use the lang-addition files and you'll add them yourself or however that works?

Edit: Question two: is the key combination Shift-Rightclick on the amulet actually shift rightclick or sneak rightclick but yknow what it doesnt matter because I'll translate it as it stands there

myron-obsolete-account commented:
Finished with 3.2 translations, 3.2.1 translations are following up

[3.2-lang-additions.txt](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12398607/3.2-lang-additions.txt)

myron-obsolete-account commented:
finished 3.2.1 translations

[3.2.1-lang-additions.txt](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12398653/3.2.1-lang-additions.txt)

@nonamecrackers2 finished update translations, you could also perhaps leave this open for 3.3 so I can then translate the next few things when the update is out.

myron-obsolete-account commented:
Oh I forgot to add that I have fixed a few mistakes in the original de_de language file aswell so here you go (this is from before 3.2, the additions are above)

[CWSM de_de fixed (I hope).txt](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12398718/CWSM.de_de.fixed.I.hope.txt)

IndigoAmethyst commented:
gut

MoRoN222 commented:
I feel like this is being ignored somehow, not on purpose though so thats fine I guess? as long as I can make him see this at some point

nonamecrackers2 commented:
If you can combine your changes with the current translation into a single file that would be most appreciated, would save me a lot of time

myron-obsolete-account commented:
if it works that easy then alright gimme a minute

myron-obsolete-account commented:
[de_de.zip](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12445321/de_de.zip)
get zipped up, json file

myron-obsolete-account commented:
wait hold on I missed out something
![image](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/19b52f47-7a32-487c-a63c-a90e4e93ac42)

myron-obsolete-account commented:
what exactly does that mean, "working area", the area where they work or the area where they... work? I hate languages

myron-obsolete-account commented:
Working area as in where they function where you place them or working area as in what areas are affected by the beacon?

myron-obsolete-account commented:
~okay nevermind I went for "Working area as in where they function where you place them"~

~[de_de.zip](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12446486/de_de.zip)~
~this is the fixed one use this instead of the other one~

mistake's been resolved, this is irrelevant

myron-obsolete-account commented:
okay I hope this is the final version istg if theres something else wrong/missing im going to explode
[the language of the germans.zip](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12448264/de_de.zip)

nonamecrackers2 committed:
Update README.md

watersofnazareth commented:
Closed due to how outdated this is
